stable_baselines/common/distributions.py

```diff
@@ -344,12 +352,17 @@ def __init__(self, nvec, flat):
         """
         self.flat = flat
         self.categoricals = list(map(CategoricalProbabilityDistribution, tf.split(flat, nvec, axis=-1)))
+        self.action_mask_vector = action_mask_vector
 
     def flatparam(self):
         return self.flat
 
     def mode(self):
```

Comment:
I decided to make the action mask a "layer" in the policy, as opposed to it being done in distribution, which caused those particular tests to fail as they don't invoke a policy. I like this way a lot more, as it's much cleaner IMO.

Reply:
yes, sounds good